File: COM_SIC_Procesa_Pagos/DataPowerRest/BEAuditRequest.cs
using System;

namespace COM_SIC_Procesa_Pagos.DataPowerRest
{
	
	public class BEAuditRequest
	{
		private string _idTransaccion;
		private string _ipAplicacion ; 
		private string _nombreAplicacion;
		private string  _usuarioAplicacion;


		public string idTransaccion
		{
			set{_idTransaccion= value;}
			get{ return _idTransaccion;}
		}

		public string ipAplicacion
		{
			set{_ipAplicacion= value;}
			get{ return _ipAplicacion;}
		}

		public string nombreAplicacion
		{
			set{_nombreAplicacion= value;}
			get{ return _nombreAplicacion;}
		}

		public string usuarioAplicacion
		{
			set{_usuarioAplicacion= value;}
			get{ return _usuarioAplicacion;}
		}
				

		public BEAuditRequest()
		{
			
		}
	}
}

File: COM_SIC_Activaciones/RegistrarCampaniaRest/BEAuditRequest.vb
'INC000002161718 inicio
Imports System

Public Class BEAuditRequest

    Private _idTransaccion As String
    Private _ipAplicacion As String
    Private _nombreAplicacion As String
    Private _usuarioAplicacion As String

    Public Property idTransaccion() As String
        Get
            Return _idTransaccion
        End Get
        Set(ByVal Value As String)
            _idTransaccion = Value
        End Set
    End Property

    Public Property ipAplicacion() As String
        Get
            Return _ipAplicacion
        End Get
        Set(ByVal Value As String)
            _ipAplicacion = Value
        End Set
    End Property
    Public Property nombreAplicacion() As String
        Get
            Return _nombreAplicacion
        End Get
        Set(ByVal Value As String)
            _nombreAplicacion = Value
        End Set
    End Property
    Public Property usuarioAplicacion() As String
        Get
            Return _usuarioAplicacion
        End Get
        Set(ByVal Value As String)
            _usuarioAplicacion = Value
        End Set
    End Property

End Class
